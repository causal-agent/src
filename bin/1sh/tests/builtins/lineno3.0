# $FreeBSD: releng/12.1/bin/sh/tests/builtins/lineno3.0 272482 2014-10-03 20:24:56Z jilles $

echo before: $LINENO
dummy=$'a\0
'
echo after: $LINENO
