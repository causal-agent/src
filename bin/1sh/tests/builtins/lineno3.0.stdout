before: 3
after: 6
